docs/owo/setup.md

```diff
@@ -34,13 +34,34 @@ Then, declare the dependency inside your `dependencies` block and as well as the
 === "build.gradle (Neoforge)"
     ```groovy 
     dependencies {
-        // Moddev Projects - Neoforge
+        // Vanilla/Moddev Projects - Neoforge
```

Comment:
What does vanilla mean in this regard? If it refers to ModDevGradles Vanilla-Mode, then it should probably be specified like so:
```diff
+ // ModDevGradle (including Vanilla-Mode) - NeoForge

Reply:
I think it refers to such but there is also [VanillaGradle](https://github.com/SpongePowered/VanillaGradle) so 🤷

Reply:
I don't think that is something we can support, as owo-lib depends on FAPI/FFAPI

Reply:
Given my understanding this is in reference to something different, you get to decide on whether we change it. Either is fine :+1: